### PR TITLE
it prints hello world five times

### PR DESCRIPTION
**Pull request by Dansolo1235**
I made it print 5 times in a row

### CODE REVIEW

File: hello.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 public class hello{
     public static void main(String[] args) {
-    System.out.println("hello world");
```
Comment (Samueljdk) [inline hello.java:3]: why did you delete this code

Comment (Dansolo1235) [reply hello.java:3]: I just think printing the code 5 times is cooler

Comment (Samueljdk) [reply hello.java:3]: whatever

**Review by Samueljdk — APPROVED** [review]
